src/axom/quest/detail/PointFinder.hpp

```diff
@@ -351,6 +352,11 @@ class PointFinder
     return m_cellBBoxes[cellIdx];
   }
 
+  std::vector<IndexType> getCandidates(SpacePoint const& pt) const
+  {
+    return m_grid.getCandidatesAsArray(pt);
```

Comment:
Note: `ImplicitGrid` returns a `std::vector` for its `getCandidatesAsArray`. We should probably change this to an `axom::Array` so it's amenable w/ usage on a device.